cmd/minikube/cmd/mount.go

```diff
@@ -186,6 +186,12 @@ var mountCmd = &cobra.Command{
 			cfg.Options[parts[0]] = parts[1]
 		}
 
+		// check if linux host supports file system 9p
+		if runtime.GOOS == "linux" {
+			if !detect.IsNinePSupported() {
```

Comment:
Could we combine these ifs with `&&` since there's no else case for the nested if